waspc/data/Generator/templates/sdk/client/index.ts

```diff
@@ -0,0 +1,9 @@
+import * as path from 'path';
+
+/**
+ * Make a pattern relative from Wasp's web-app directory to the project's root.
+ * This enables users to define a glob pattern that is relative to the project's root.
+ */
+export function makeProjectGlobPattern(globPattern: string): string {
```

Comment:
Users can use this helper when they are defining their glob patterns in the Tailwind config.

Alternatively, we could make a `createTailwindConfig` helper that could go through the `content` array and do this for them? I'm not sure which API has the better DX.

Reply:
I think `createTailwindConfig` would be a bit much.

I think we might even go in the opposite direction and just offer them `waspProjectRootPath: string`? Then they have even more freedom when using that (it is not specific to globs).

So they would do `waspProjectRootPath + 'src/...'`.

Yeah, this sounds potentially the best to me, although if you don't like it, I am ok with sticking with current solution.

Reply:
We could rename the helper to say `resolveProjectPath` or similar that would make it more general.

In the Tailwind specific docs we show them how to use our project path helper. And users can use that helper where ever else which we will find out when people start asking questions 😄

Reply:
Hah yes, why doesn't it work :D.
What I meant is that if we give them string instead of function, and we remove the `glob` word from it, then this thisn we exposed as an API will be more robust to not changing so quickly, which I thought could be beneficial. Yes, there is a risk of them using it incorrectly though, but for some reason that doesn't sound so scary to me.

OK, with this I will leave you to make the final decision on your own.
I would probably go with just a string that has no `glob` in the name, if not then the second option I would go with is function as you have without the `glob` in the name, and the last (but acceptable) option for me is what we currently have which is a function with `glob` in the name.